src/components/my-component/my-component.tsx

```diff
@@ -1,74 +1,87 @@
+// Import necessary modules and libraries
 import { Component, Prop, h, Element, Watch } from '@stencil/core';
-import * as d3 from 'd3';  // Import D3.js
+import * as d3 from 'd3';  // Import D3.js for data visualisation
```

Comment:
Sooner or later, including only used parts of D3 (as soon as it is clear what is needed) might be a better choice to keep the component small.